projectq/meta/_compute.py

```diff
@@ -87,6 +87,21 @@ def _add_uncompute_tag(self, cmd):
         cmd.tags.append(UncomputeTag())
         return cmd
 
+    def _send_rewritten_command(self, cmd, rewrite_id_map):
```

Comment:
Why is this part of the old (arguable long) function moved into a private function?
It is only called at one place. This only adds another function call overhead and for understanding run_uncompute, I have to scroll up to check what this function does...